client/src/components/ui/pagination.tsx

```diff
@@ -0,0 +1,58 @@
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+
+type PaginationProps = {
+  currentPage: number;
+  totalPages: number;
+  onPageChange: (page: number) => void;
+}
+
+function PaginationArrow({ isDisabled, onClick, children, ...props }: { isDisabled: boolean; onClick: () => void; children: React.ReactNode }) {
+  return (
+    <button
+      disabled={isDisabled}
+      className={`flex justify-center items-center h-[32px] w-[32px] border-[2px] border-ui-hover rounded-[4px] bg-light cursor-pointer disabled:cursor-not-allowed ${!isDisabled ? "hover:bg-ui-hover" : ""}`}
+      onClick={onClick}
+      {...props}
+    >
+      {children}
+    </button>
+  )
+}
+
+function Pagination({ currentPage, totalPages, onPageChange }: PaginationProps) {
+
+  const pageNumbers = Array.from({ length: totalPages }, (_, i) => i + 1);
+
+  return (
+    <div className='flex justify-center items-center gap-[4px]'>
+      <PaginationArrow
+        isDisabled={currentPage === 1}
+        onClick={() => onPageChange(currentPage - 1)}
+      >
+        <ChevronLeftIcon sx={{
+          fill: currentPage === 1 ? "#EAEAEA" : "#2E2E2E"
```

Comment:
🌴 🌴 🌴 Non-blocking comment: We could reference our site-wide css variables here:
```suggestion
          fill: currentPage === 1 ? "var(--color-gray-3)" : "var(--color-gray-1)"
```